Helper/PCA Helper.md

```diff
@@ -0,0 +1,105 @@
+# PCA Helper
+PCA developers use common pattern to acquire token. The code appears to be repeatative and very granular. At the same time, the current API is based on the builder pattern. There are several "With" APIs and a some are used in commonly in the Public Client Application (PCA). Due to the abudance of the With APIs, the learning curve can be high to perform simple/common tasks.
+
+PCAHelper extracts API at a higher level offering flexibility for granular programming as desired. The helper provides methods with optional paramaeters and lambdas for customization. It has the following features:
+
+- It comes with two main common practices as default:
+    - There is only one instance of PCA
+    - Scope for permissions is defined only once.
+    The above default can be customized as desired.
+ - It is inteface based. So the calling app can create Mocks and perform testing w/o havinng dependency on MSAL.NET and any network connectivity
+ - It allows customization of token acqusition methods.
+ - It allows specialization of the Helper class should developer choose it.
+
+# APIs
+The APIs are briefly described here.
+
+## Initialization
+This can be done once in the App. This initializes the helper with client id and scope, if it does not have a standrd redirect URI, it can be customized here. By default it PCAHelper is singleton. It can be overwritten by forcing creation.
```

Comment:
```suggestion
This can be done once in the App. This initializes the helper with client id and scope, if it does not have a standard redirect URI, it can be customized here. By default the PCAHelper is a singleton. It can be overwritten by forcing creation.
```